todo_app/data/view_model.py

```diff
@@ -0,0 +1,19 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+ 
+    @property
+    def open_items(self):
+        output = []
+        for item in self._items:
+            if item.status == 'Open':
+                output.append(item)
+        return output
```

Comment:
This works nicely, so the below isn't a suggestion, but I do think is worth a comparison - list comprehensions can often allow us to write more concise code, but potentially at the cost of being harder to read. What do you prefer?
```python
        return [item for item in self._items if item.status == 'Open']
```